packages/website/docs/api.mdx

```diff
@@ -118,11 +117,17 @@ An implementation of [Algolia Autocomplete][1]’s Navigator API to redirect the
 
 Learn more on the [Navigator API][11] documentation.
 
-</TabItem>
+## `translations`
 
+> `type: Partial<DocSearchTranslations>` | `default: docSearchTranslations` | **optional**
 
-<TabItem value="react">
+Allow translations of any raw text present in the DocSearch button or modal components.
+
+See the [default implementation](https://github.com/algolia/docsearch/blob/next/packages/docsearch-react/src/DocSearch.tsx#L20-L23)
```

Comment:
The link is not working yet since it's not deployed on master, it will point to: https://github.com/algolia/docsearch/blob/2b7197fedf69ca08af95601fdd116061c20835da/packages/docsearch-react/src/DocSearch.tsx#L20-L23

Do you think we should write them all in the doc?

Reply:
yep I figured out this part but it's not good enough, even with typescript on it's hard to find where the string is since it's highly nested

Reply:
Added in https://github.com/algolia/docsearch/pull/1183/commits/86ba124b3312b73861e4caeb4308e16e5962555c, see preview https://deploy-preview-1183--docsearch.netlify.app/docs/api/#translations